<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerContourMorphologyModule</class>
 <widget class="qSlicerWidget" name="qSlicerContourMorphologyModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contour Morphology</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Parameter set:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ParameterSet" native="true">
       <property name="nodeTypes" stdset="0">
        <stringlist>
         <string>vtkMRMLContourMorphologyNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton_Input" native="true">
     <property name="text" stdset="0">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Contour A:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Contour B:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLContourSelectorWidget" name="ContourSelectorWidget_ContourA" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLContourSelectorWidget" name="ContourSelectorWidget_ContourB" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_OperationParameters" native="true">
     <property name="text" stdset="0">
      <string>Operation parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Expand or Shrink in LR (mm):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Expand or Shrink in AP (mm):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Expand or Shrink in SI (mm):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_ZSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_YSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_XSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="radioButton_Shrink">
        <property name="text">
         <string>Shrink A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButton_Expand">
        <property name="text">
         <string>Expand A</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../qSlicerContourMorphologyModule.qrc">:/Icons/expand.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../qSlicerContourMorphologyModule.qrc">:/Icons/shrink.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="radioButton_Union">
        <property name="text">
         <string>A U B</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="radioButton_Intersect">
        <property name="text">
         <string>A n B</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QRadioButton" name="radioButton_Subtract">
        <property name="text">
         <string>A - B</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../qSlicerContourMorphologyModule.qrc">:/Icons/union.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../qSlicerContourMorphologyModule.qrc">:/Icons/intersect.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../qSlicerContourMorphologyModule.qrc">:/Icons/subtract.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Select operation:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_Output" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text" stdset="0">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_OutputContour">
        <property name="text">
         <string>Output contour:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_OutputContour" native="true">
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLContourNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="noneDisplay" stdset="0">
         <string>Create New Contour</string>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Apply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLContourSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLContourSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerContourMorphologyModule.qrc"/>
  <include location="../qSlicerContourMorphologyModule.qrc"/>
  <include location="../qSlicerContourMorphologyModule.qrc"/>
  <include location="../qSlicerContourMorphologyModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerContourMorphologyModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ParameterSet</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerContourMorphologyModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_OutputContour</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerContourMorphologyModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ContourSelectorWidget_ContourA</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerContourMorphologyModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ContourSelectorWidget_ContourB</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
